src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -961,4 +961,7 @@ This version used in: {2}</value>
   <data name="AddMissingSwitchCases" xml:space="preserve">
     <value>Add missing switch cases</value>
   </data>
+  <data name="Make_method_synchronous_changing_return_type" xml:space="preserve">
```

Comment:
Cyrus is trying to change us one resource at a time. :smile:

I actually like this, but I also implied in another review that I'm waiting for him to convert all of the resources (after getting consensus from the rest of the team of course!).